SPECS-EXTENDED/buildah/buildah.spec

```diff
@@ -21,7 +21,7 @@
 Summary:        A command line tool used for creating OCI Images
 Name:           buildah
 Version:        1.18.0
-Release:        24%{?dist}
+Release:        25%{?dist}
```

Comment:
It seems we have missed to add a changelog entry for the release bump. (for all the specs)